### PR TITLE
Update recommendation.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated missing properties

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5910

File: microsoft-365/security/defender-endpoint/recommendation.md [hunk 1]
```diff
@@ -53,7 +53,9 @@ recommendationName | String | Recommendation name
 Weaknesses | Long | Number of discovered vulnerabilities
 Vendor | String | Related vendor name
 recommendedVersion | String | Recommended version
-recommendationCategory | String | Recommendation category. Possible values are: "Accounts", "Application", "Network", "OS", "SecurityStack
+recommendedProgram | String | Recommended Program
+recommendedVendor | String | Recommended Vendor
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/recommendation.md]:
```suggestion
recommendedProgram | String | Recommended program
recommendedVendor | String | Recommended vendor
```

File: microsoft-365/security/defender-endpoint/recommendation.md [hunk 2]
```diff
@@ -62,7 +64,7 @@ associatedThreats | String collection | Threat analytics report is associated wi
 remediationType | String | Remediation type. Possible values are: "ConfigurationChange","Update","Upgrade","Uninstall"
 Status | Enum | Recommendation exception status. Possible values are: "Active" and "Exception"
 configScoreImpact | Double | Microsoft Secure Score for Devices impact
-exposureImpacte | Double | Exposure score impact
+exposureImpact | Double | Exposure score impact
 totalMachineCount | Long | Number of installed devices
 exposedMachinesCount | Long | Number of installed devices that are exposed to vulnerabilities
 nonProductivityImpactedAssets | Long | Number of devices which are not affected
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/recommendation.md]:
```suggestion
nonProductivityImpactedAssets | Long | Number of devices that are not affected  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes for consistency.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@siosulli Gentle reminder

**Pull request merged**